<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QOperationStateScreenBase</class>
 <widget class="QWidget" name="udg::QOperationStateScreenBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Queue Operation List</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="3" column="5">
    <widget class="QPushButton" name="m_cancellAllRequestsButton">
     <property name="text">
      <string>Cancel All</string>
     </property>
     <property name="icon">
      <iconset resource="../main/themes/starviewer/starviewer.qrc">
       <normaloff>:/images/icons/process-stop.svg</normaloff>:/images/icons/process-stop.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="4">
    <widget class="QTreeWidget" name="m_treeRetrieveStudy">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Status</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Direction</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>From/To</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Patient ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Patient Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Request Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Started</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Series</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Files</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="m_cancelSelectedRequestsButton">
     <property name="text">
      <string>Cancel Selected</string>
     </property>
     <property name="icon">
      <iconset resource="../main/themes/starviewer/starviewer.qrc">
       <normaloff>:/images/icons/process-stop.svg</normaloff>:/images/icons/process-stop.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="m_buttonClear">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>32767</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
     <property name="icon">
      <iconset resource="../main/themes/starviewer/starviewer.qrc">
       <normaloff>:/images/icons/edit-clear.svg</normaloff>:/images/icons/edit-clear.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_treeRetrieveStudy</tabstop>
 </tabstops>
 <resources>
  <include location="../main/themes/starviewer/starviewer.qrc"/>
 </resources>
 <connections/>
</ui>
